manifests/server.pp

```diff
@@ -15,7 +15,7 @@
 #
 class rsyslog::server (
   Boolean $enable_firewall    = simplib::lookup('simp_options::firewall', { 'default_value' => false }),
-  Boolean $enable_selinux     = $facts['selinux_enforced'],
+  Boolean $enable_selinux     = str2bool($facts['selinux_enforced']),
```

Comment:
@nick-markowski Looking at the bug in JIRA, this should be changed to an `Optional` parameter since `undef` and `false` will both work appropriately with the logic. We should not use `str2bool` unless it's 100% necessary.